doc/nrf/known_issues.rst

```diff
@@ -799,28 +800,33 @@ KRKNWK-6073: Potential delay during FOTA
 Matter (Project CHIP)
 =====================
 
-.. rst-class:: v1-9-1 v1-9-0
-
-KRKNWK-12950: CHIP Tool for Android opens the commissioning window using an incorrect PIN code
-  CHIP Tool for Android uses a random code instead of a user-provided PIN code to open the commissioning window on a Matter device.
+.. rst-class:: v2-0-0
 
+KRKNWK-14180: The QSPI sleep mode is not handled efficiently in Matter samples on the nRF53 SoC
+  QSPI is active during every Bluetooth LE connection in the Matter samples that are programmed on the nRF53 SoC.
+  This results in higher power consumption, for example during commissioning into the Matter network.
```

Comment:
```suggestion
  This results in higher power consumption, for example, during commissioning into the Matter network.
```

Reply:
This might slightly change the meaning, and besides this usage of for example without the following comma is pretty common in our docs, so I'll skip this edit.